utils/node_local_test.py

```diff
@@ -6139,7 +6139,9 @@ def get_cmd(cont_id):
                 '--type',
                 'POSIX',
                 '--path',
-                join(dfuse.dir, f'container_{cont_id}')]
+                join(dfuse.dir, f'container_{cont_id}'),
+                '--properties',
+                f'srv_cksum:on,label:{cont_id}']
```

Comment:
These are properties, not attributes I think?  Good to have but they're not the new feature in this PR?  I would also say this is nice test coverage to have but I'd also be OK with landing the feature and adding this testing in a later PR.

Reply:
> * [x]  add attributes

Fixed with commit 2cb2ddd